db/seeds.rb

```diff
@@ -761,6 +761,59 @@ def report_list
         health: false,
       },
     ],
+    'High Level Dashboards' => [
```

Comment:
We lost some dynamism here vs the old one that itemizes by the installed drivers... Is that OK?

Reply:
Good catch, I'm getting sloppy.  Pushed a fix.